sys/net/include/rpl/rpl_dodag.h

```diff
@@ -26,6 +26,10 @@
 extern "C" {
 #endif
 
+extern ipv6_addr_t mcast;
+
+void dao_ack_received(rpl_dodag_t *dodag);
+void delay_dao(rpl_dodag_t *dodag);
```

Comment:
I know it doesn't hold for the other functions, but please add prefix here, too.